enterprise/backend/test/metabase_enterprise/serialization/v2/e2e/yaml_test.clj

```diff
@@ -346,3 +358,96 @@
                (is (= (into {} (for [{:keys [key value]} (get @entities "Setting")]
                                  [key value]))
                       (yaml/from-file (io/file dump-dir "settings.yaml"))))))))))))
+
+(deftest card-and-dashboard-has-parameter-with-source-is-card-test
+  (testing "Dashboard and Card that has parameter with source is a card must be deserialized correctly"
```

Comment:
This is the only e2e test that doesn't use specmonstah to generate example data. Can you leave a note explaining why?